index.js

```diff
@@ -553,8 +539,8 @@ export async function timer({ key }) {
   }
 
   // Schedule the next check
-  const timer = Number.parseInt(meta['x-poll-interval']);
-  await program.setTimer(key, 0, timer);
```

Comment:
This should be `setTimer(key, eventTimer)`. First argument is delay, second is frequency. We don't want this timer to repeat so it should have no `frequency`

Reply:
Also, you might want to rename `eventTimer` to `pollInterval`